### PR TITLE
Add System.Console to netcoreapp.props files

### PR DESCRIPTION
**Pull request by wtgodbe**
@safern PTAL

### CODE REVIEW

File: src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props [hunk 1]
```diff
@@ -57,6 +57,9 @@
     <NETCoreApp11Dependency Include="System.ComponentModel.Annotations">
       <Version>4.3.0</Version>
     </NETCoreApp11Dependency>
+    <NETCoreApp10Dependency Include="System.Console">
```
Comment (safern) [inline src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props]: This should be NETCoreApp11Dependency

**Comment by safern** [comment]
Also in 1.1 branch I don't see System.Console included in the NETCoreApp project.json file

https://github.com/dotnet/core-setup/blob/release/1.1.0/pkg/projects/Microsoft.NETCore.App/project.json

You will need to update that before servicing, if not the System.Console will not be picked up in the closure in NETCoreApp v1.1.8

**Comment by safern** [comment]
> Also in 1.1 branch I don't see System.Console included in the NETCoreApp project.json file

I see you're doing this in: https://github.com/dotnet/core-setup/pull/3868/files#diff-d42418420888e3555f05ef5c16792861R20

Sorry about that.

File: src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props (hunk 1 above)
Comment (wtgodbe) [reply src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props]: My bad - copy/paste error. Fixed.

**Pull request merged**